### ISSUE TITLE
[Bug Report]: When the plugin uses 'esp-idf master', there is a problem with the monitoring function, but 'esp-idf v4.4' is fine (VSC-889)

### ISSUE THREAD

**Issue by xiaguangbo** [espressif/vscode-esp-idf-extension#699]
Hi
When the plugin uses 'esp-idf master', there is a problem with the monitoring function, but 'esp-idf v4.4' is fine
I've tried back and forth several times
I don't know if the plugin version has not kept up or there is a problem with esp-idf...

It seems that the 'IDF_TOOLS_PATH' variable cannot be found, and there is no translation when printing the variable in the terminal, but there is 'IDF_PATH'. After I manually introduce it in the terminal, it can run by manually executing the monitoring command.

![QQ截图20220419122704](https://user-images.githubusercontent.com/102402151/163920609-60de30a1-a0fb-42a7-90d5-52152baa7e63.png)

I checked a lot of places, including the idf part set by the vscode user, the environment variable part of the vscode user configuration, and the idf log, but I didn't find any abnormality.

**Info**

```
PS C:\Users\xiaguangbo\esp\project\sample_project> C:\Users\xiaguangbo\esp\master\.espressif\python_env\idf5.0_py3.8_env\Scripts\python.exe C:\Users\xiaguangbo\esp\master\esp-idf\tools\idf.py -p COM6 monitor
ERROR: C:\Users\xiaguangbo\.espressif\python_env\idf5.0_py3.8_env\Scripts\python.exe doesn't exist! Please run the install script or "idf_tools.py install-python-env" in order to create it

ESP-IDF v5.0-dev-2483-g3aeb80acb6-dirty
```

**Environment**

- OS Name & Version: win11
- VSCode Version: 1.66.2
- ESP-IDF Version: 1.4
- Python Version: 3.8.7

**Comment by brianignacio5** [comment]
Sorry for late reply @xiaguangbo I think i found the issue.

The extension was not following the constraints given by ESP-IDF. This should be fixed in #700 and this [vsix installer](https://nightly.link/espressif/vscode-esp-idf-extension/actions/artifacts/216432431.zip).

**Comment by xiaguangbo** [comment]
@brianignacio5 thanks
